scripts/metavar.py

```diff
@@ -1823,6 +1865,32 @@ def __eq__(self, other):
         list equality"""
         return self is other
 
+    @classmethod
+    def unit_cnv_match(cls, hunits, var):
+        """Return VarUnitConv if <hunits> and <var> require unit
+        conversion, otherwise, return None"""
+        ucmatch = {"from": '' ,"to": ''}
+        function_name_from = '{0}__to__{1}'.format(hunits, var.get_prop_value('units'))
+        function_name_to   = '{0}__to__{1}'.format(var.get_prop_value('units'), hunits)
+
+        try:
+            function_from   = getattr(unit_conversion, function_name_from)
+            ucmatch["from"] = VarUnitConv(function_from(),key="from",
+                                          local_name=var.get_prop_value('local_name'),
+                                          kind=var.get_prop_value('kind'))
+        except:
```

Comment:
I think we want to generate an error and abort if a requested unit conversion is not implemented.

Reply:
Agreed. I was waiting until things were more settled before adding in error checking